task_7/filter-practice.js

```diff
@@ -0,0 +1,6 @@
+function filterPractice(niz) {
+  const noviNiz = niz.filter((v, i) => niz.indexOf(v)==i);
```

Comment:
Ovde mozes odma samo `return` da stavis i znak jedankosti gledaj da uvek bude trostruko `===` ako moze 
```suggestion
  return niz.filter((v, i) => niz.indexOf(v)===i);
```